### PR TITLE
AND-165 changed the status bar text color so it is in contrast with t…

### PR DESCRIPTION
**Pull request by vandus**
…he background

device-2020-11-18-203738

### CODE REVIEW

File: app/src/main/res/values-night/colors.xml [hunk 1]
```diff
@@ -11,6 +11,8 @@
 
     <color name="toolbarColor">@android:color/black</color>
     <color name="toolbarActiveColor">@android:color/white</color>
-    <color name="statusbarColor">@android:color/black</color>
     <color name="launch_screen_background">#001829</color>
+
+    <color name="statusbarColor">@android:color/black</color>
```
Comment (davidvavra) [inline app/src/main/res/values-night/colors.xml:16]: Please add also screenshot from dark mode

**Comment by davidvavra** [comment]
@vandac Screenshot didn't upload

**Comment by vandus** [comment]
Screenshot 2020-11-19 at 17 34 36

**Review by davidvavra — APPROVED** [review]
Thanks

**Pull request merged**